Brokkr/src/main/include/subsystems/Arm.h

```diff
@@ -35,6 +36,7 @@ class Arm : public frc2::SubsystemBase {
  WPI_TalonFX& ArmMotor1;
  WPI_TalonFX& ArmMotor2;
  WPI_TalonFX& TelescopingArm;
+ WPI_CANCoder& CanCoder;
```

Comment:
All of these member variables should be prefixed with an `m`. Lots of resulting changes with the renaming. Perhaps do the renaming in a separate commit.

Reply:
Fix later.